lib/_http_client.js

```diff
@@ -504,19 +504,17 @@ ClientRequest.prototype._deferToConnect = function(method, arguments_, cb) {
   // in the future (when a socket gets assigned out of the pool and is
   // eventually writable).
   var self = this;
+  var callSocketMethod = function() {
+    if (method) {
+      self.socket[method].apply(self.socket, arguments_);
+    }
+    if (cb) { cb(); }
```

Comment:
Are the absence of curly braces preferred for these one line if statements?

I'm just wondering because I've seen plenty of cases of both in this file, and I'd like to conform to consistency.